Drop-In/src/main/java/com/braintreepayments/api/SupportedPaymentMethodsAdapter.java

```diff
@@ -10,27 +11,31 @@
 import android.widget.TextView;
 
 import com.braintreepayments.api.dropin.R;
+import com.braintreepayments.cardform.utils.CardType;
 
 import java.util.List;
 
 class SupportedPaymentMethodsAdapter extends BaseAdapter {
 
-    private final List<DropInPaymentMethodType> supportedPaymentMethods;
-    private final PaymentMethodSelectedListener selectedListener;
+    private final Context mContext;
```

Comment:
Is there a reason for Hungarian notation in these new variable names?

Reply:
Actually not sure here. We can remove that.